### PR TITLE
[W07H03] Added canEat tests that checks if instanceof has been used and added move method tests

### PR DESCRIPTION
**Pull request by HendrikHuebner**

### CODE REVIEW

**Comment by 0xYelshayeb** [comment]
lgtm

File: w07h03/test/pgdp/sim/MoveMethodTest.java [hunk 1]
```diff
@@ -0,0 +1,197 @@
+package pgdp.sim;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
```
Comment (nilsreichardt) [inline w07h03/test/pgdp/sim/MoveMethodTest.java:6]: This import is unused, right?

**Review by 0xYelshayeb — COMMENTED** [review]
approved

**Review by MaximilianAnzinger — COMMENTED** [review]
🐧

**Pull request merged**